[2025-01-28T04:24:12.692+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T04:24:12.761+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo.airflow scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2025-01-28T04:24:12.792+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo.airflow scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2025-01-28T04:24:12.793+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-28T04:24:12.931+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): airflow> on 2022-02-07 00:00:00+00:00
[2025-01-28T04:24:13.060+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'demo', '***', 'scheduled__2022-02-07T00:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/documentation_dag.py', '--cfg-path', '/tmp/tmppu2lstbs']
[2025-01-28T04:24:13.067+0000] {standard_task_runner.py:105} INFO - Job 85: Subtask ***
[2025-01-28T04:24:13.081+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=263) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-28T04:24:13.082+0000] {standard_task_runner.py:72} INFO - Started process 289 to run task
[2025-01-28T04:24:14.101+0000] {task_command.py:467} INFO - Running <TaskInstance: demo.airflow scheduled__2022-02-07T00:00:00+00:00 [running]> on host a845ebe66097
[2025-01-28T04:24:14.383+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo' AIRFLOW_CTX_TASK_ID='***' AIRFLOW_CTX_EXECUTION_DATE='2022-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-07T00:00:00+00:00'
[2025-01-28T04:24:14.385+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-28T04:24:14.385+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-28T04:24:14.386+0000] {logging_mixin.py:190} INFO - Current task name:*** state:running start_date:2025-01-28 04:24:12.763500+00:00
[2025-01-28T04:24:14.391+0000] {logging_mixin.py:190} INFO - Dag name:demo and current dag run status:running
[2025-01-28T04:24:14.391+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T04:24:14.392+0000] {logging_mixin.py:190} INFO - ***
[2025-01-28T04:24:14.393+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-28T04:24:14.453+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T04:24:14.462+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=demo, task_id=***, run_id=scheduled__2022-02-07T00:00:00+00:00, execution_date=20220207T000000, start_date=20250128T042412, end_date=20250128T042414
[2025-01-28T04:24:14.493+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-28T04:24:14.499+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-28T04:24:14.500+0000] {logging_mixin.py:190} INFO - Dag name:demo queued_at:2025-01-28 04:23:55.136984+00:00
[2025-01-28T04:24:14.501+0000] {logging_mixin.py:190} INFO - Task hostname:a845ebe66097 operator:_PythonDecoratedOperator
[2025-01-28T04:24:14.576+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T04:24:15.232+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T04:24:15.235+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
